src/app/api/api_v1/endpoints/detections.py

```diff
@@ -128,6 +132,40 @@ async def fetch_detections(
     return filtered_detections
 
 
+@router.get("/unacknowledged/from", status_code=status.HTTP_200_OK, summary="Fetch all the unacknowledged detections")
+async def fetch_unacknowledged_detections(
+    from_date: Optional[datetime] = None,
+    detections: DetectionCRUD = Depends(get_detection_crud),
+    cameras: CameraCRUD = Depends(get_camera_crud),
+    token_payload: TokenPayload = Security(get_jwt, scopes=[UserRole.ADMIN, UserRole.AGENT, UserRole.USER]),
+) -> List[Detection]:
+    telemetry_client.capture(token_payload.sub, event="unacknowledged-fetch")
+
+    try:
+        all_unck_detections = [elt for elt in await detections.fetch_all() if elt.is_wildfire is None]
+        if from_date is not None:
+            all_unck_detections = [detection for detection in all_unck_detections if detection.created_at >= from_date]
+    except Exception as e:  # noqa
+        logging.error(f"Error fetching detections: {e}")
+        raise HTTPException(status_code=500, detail="Internal server error")
+
+    if UserRole.ADMIN in token_payload.scopes:
+        return all_unck_detections
+
+    filtered_detections = []
+    for detection in all_unck_detections:
+        try:
+            camera = await cameras.get(detection.camera_id, strict=True)
+        except Exception as e:  # noqa
```

Comment:
Either we let strict=True throw an error or we set strict=False and check whether it's None to avoid the try except